.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale pull requests"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: "This pull request is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days."
+        days-before-stale: 30
+        days-before-close: 5
```

Comment:
can we set this to -1 to start, so that issues and PRs are just labelled in the first instance and not closed?